Revert/Sources/CollectionViewListController.swift

```diff
@@ -0,0 +1,89 @@
+// Copyright © 2020 Itty Bitty Apps. All rights reserved.
+
+import UIKit
+
+final class CollectionViewListController: RevertCollectionViewController {
+  fileprivate enum Section: CaseIterable {
+    case countries
+  }
+
+  fileprivate let countries = Country.countries
+
+  @available(iOS 14.0, *)
+  fileprivate lazy var dataSource: UICollectionViewDiffableDataSource<Section, Country> = {
+    let cellRegistration = UICollectionView.CellRegistration<UICollectionViewListCell, Country> { cell, indexPath, country in
+      var content = cell.defaultContentConfiguration()
+      content.text = country.name
+      content.secondaryText = country.capital
+
+      cell.contentConfiguration = content
+    }
+
+    return UICollectionViewDiffableDataSource<Section, Country>(collectionView: collectionView) { (collectionView, indexPath, country) -> UICollectionViewCell? in
+      collectionView.dequeueConfiguredReusableCell(using: cellRegistration, for: indexPath, item: country)
+    }
+  }()
+
+  override func viewDidLoad() {
+    if #available(iOS 14.0, *) {
+      configureLayout()
+      applySnapshot()
+    }
+  }
+}
+
+// MARK: - iOS 13 and below configuration
+extension CollectionViewListController: UICollectionViewDelegateFlowLayout {
+  override func numberOfSections(in collectionView: UICollectionView) -> Int {
+    return Section.allCases.count
+  }
+
+  override func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+    return self.countries.count
+  }
+
+  override func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+    guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "CollectionViewListCell", for: indexPath) as? CollectionViewListCell else {
+      fatalError("Expecting to dequeue a \(CollectionViewListCell.self) from the collection view")
+    }
+    cell.label.text = countries[indexPath.row].name
+
+    return cell
+  }
```

Comment:
Removing this will crash when you run on iOS 13 and below. The purpose of this configuration is to show a cell that displays a label that says "Collection View List only available on iOS 14 and above "